package.json

```diff
@@ -20,12 +20,13 @@
     "changelog:update": "yarn workspaces foreach --all --no-private --parallel --interlaced --verbose run changelog:update",
     "changelog:validate": "yarn workspaces foreach --all --no-private --parallel --interlaced --verbose run changelog:validate",
     "create-package": "ts-node scripts/create-package",
+    "postinstall": "cd apps/storybook-react-native/ios && pod install || exit 0",
```

Comment:
Shout this be named `postinstall` or `podinstall`?

Reply:
postinstall

Reply:
I think the naming of this script is causing the lavamoat issue in [this comment](https://github.com/MetaMask/metamask-design-system/pull/60/files#r1811609620). When I rename this script I no longer get this Lavamoat issue. 


https://github.com/user-attachments/assets/bb0a91ed-5c68-4b9e-9fbe-b730b7ab2077

Why Not Set `"$root$": false`?

While it might be tempting to set `"$root$": false` in the Lavamoat configuration to quickly bypass the issue, doing so is **not recommended**. Here's why:

- **Disallows All Root Scripts:** Setting `"$root$": false` effectively prevents any scripts from running at the root level of the project. This blanket restriction can inadvertently block legitimate and necessary scripts, including `postinstall`, leading to further setup and functionality issues.

- **Overly Restrictive:** Rather than selectively permissive, this approach is too broad and doesn't allow for the flexibility needed to run specific scripts that are essential for the project's setup and maintenance.

- **Security Implications:** Disabling scripts at the root level entirely may lead to scenarios where necessary scripts are not executed, potentially breaking the build or deployment processes. It also doesn't provide a secure way to manage which scripts are allowed, reducing the overall effectiveness of Lavamoat's security measures.

Reply:
`yarn pod:install` still works

![Screenshot 2024-10-23 at 12 03 08 PM](https://github.com/user-attachments/assets/961cf8e4-409d-444f-8829-d518acf0d140)